src/app/offers/offers-card-large/offers-card-large.component.html

```diff
@@ -40,8 +44,11 @@ <h1 class="mt-6 lg:mt-0 text-4xl uppercase tracking-wider font-light leading-rel
         {{ offer.startDate | date: 'mediumDate'}} - {{ offer.endDate | date: 'mediumDate'}}
       </div>
     </div>
+
+    <!-- TODO: remove hardcoded routerLink from button -->
     <button class="bg-indigo-800 hover:bg-indigo-900 transition-all
-    w-full py-3 mt-6 text-white rounded-full flex items-center justify-center">
+    w-full py-3 mt-6 text-white rounded-full flex items-center justify-center"
+            routerLink="12">
```

Comment:
It is not convenient to add a router link to the entire offer card. People sometimes want to copy the title, description, whatever.. It's a bad pattern. For now, I've set "Learn more" button and "offer image" to open the offer.